### PR TITLE
Issue regarding filter text box.

### PR DESCRIPTION
**Pull request by mokalebo**
Modified filter textbox on entry screen.

### CODE REVIEW

**Comment by djw8605** [comment]
Did you test this change?  I get a compile error:

```
(node:90339) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
Running "less:production" (less) task
>> ParseError: Unrecognised input in HCCGo/app/css/application.less on line 123, column 5:
>> 122     width: 40%;
>> 123     .box-shadow: 0px;
>> 124 }
Warning: Error compiling HCCGo/app/css/application.less Use --force to continue.

Aborted due to warnings.
```

Also, I'm not sure modifying the CSS for `card` will fix the styling issue.  I thought you would have to add some special class to the input text box.  Something like on this webpage:
https://fezvrasta.github.io/bootstrap-material-design/#input

File: HCCGo/app/css/application.less [hunk 1]
```diff
@@ -116,9 +116,9 @@ body {
 }
 
 .card {
-  .box-shadow(0 12px 15px rgba(0, 0, 0, 0.24));
-  background-color: @whiteBackground;
-  padding: 0.5em;
+ -  .box-shadow(0 12px 15px rgba(0, 0, 0, 0.24));
+ -  background-color: @whiteBackground;
+ -  padding: 0.5em;
```
Comment (djw8605) [inline HCCGo/app/css/application.less]: Why are there these "dashes"?

Comment (mokalebo) [inline HCCGo/app/css/application.less]: Not entirely sure, I copied and pasted the changes from my local environment so that might be the cause

Comment (djw8605) [inline HCCGo/app/css/application.less]: Ok, go ahead and remove the dashes.

**Comment by djw8605** [comment]
I'm going to close this.  Go ahead and open a new pull request when you are ready.

**Pull request closed without merging**